2020-12-18 15:37:57.9959 ERROR Something went wrong in GetCompanies action AutoMapper.AutoMapperMappingException: Error mapping types.

Mapping types:
Object -> IEnumerable`1
System.Object -> System.Collections.Generic.IEnumerable`1[[Entities.DataTransferObjects.CompanyDto, Entities, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]]
 ---> AutoMapper.AutoMapperMappingException: Missing type map configuration or unsupported mapping.

Mapping types:
Company -> CompanyDto
Entities.Models.Company -> Entities.DataTransferObjects.CompanyDto
   at lambda_method(Closure , Company , CompanyDto , ResolutionContext )
   at lambda_method(Closure , Object , IEnumerable`1 , ResolutionContext )
   --- End of inner exception stack trace ---
   at lambda_method(Closure , Object , IEnumerable`1 , ResolutionContext )
   at CompanyEmployees.Controllers.CompaniesController.GetCompanies()
2020-12-18 15:38:44.6140 ERROR Something went wrong in GetCompanies action AutoMapper.AutoMapperMappingException: Error mapping types.

Mapping types:
Object -> IEnumerable`1
System.Object -> System.Collections.Generic.IEnumerable`1[[Entities.DataTransferObjects.CompanyDto, Entities, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]]
 ---> AutoMapper.AutoMapperMappingException: Missing type map configuration or unsupported mapping.

Mapping types:
Company -> CompanyDto
Entities.Models.Company -> Entities.DataTransferObjects.CompanyDto
   at lambda_method(Closure , Company , CompanyDto , ResolutionContext )
   at lambda_method(Closure , Object , IEnumerable`1 , ResolutionContext )
   --- End of inner exception stack trace ---
   at lambda_method(Closure , Object , IEnumerable`1 , ResolutionContext )
   at CompanyEmployees.Controllers.CompaniesController.GetCompanies()
2020-12-18 19:52:09.5380 ERROR Something went wrong: System.Exception: hui
   at CompanyEmployees.Controllers.CompaniesController.GetCompanies() in D:\Projects\CompanyEmployees\CompanyEmployees\Controllers\CompaniesController.cs:line 31
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
